gkeonprem_baremetal_cluster_manuallb/main.tf

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[START gkeonprem_bare_metal_cluster_manuallb]
+# Fill in the local variables according to your baremetal environment; you must
```

Comment:
Can you update the sample without the use of `locals.`?

You can refer to other TF samples for reference

Reply:
Yes, please see guidance at [go/terraform-docs-samples-repo-style](http://goto.google.com/terraform-docs-samples-repo-style).

Reply:
Can I get access to that doc? I cannot view it

Reply:
The content has been ported to the following locations:

* https://googlecloudplatform.github.io/samples-style-guide/ 
* https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#review-terraform-sample-requirements-best-practices-and-code-style